wolfcrypt/src/asn.c

```diff
@@ -3884,8 +3605,17 @@ int wc_EncryptPKCS8Key(byte* key, word32 keySz, byte* out, word32 outSz,
     if (ret == 0) {
         /* outer = SEQ [ pbe ] OCT encrypted_PKCS#8_key */
         outerLen = 2 + pbeLen;
-        outerLen += SetOctetString(keySz + padSz, out);
         outerLen += keySz + padSz;
+        if (out == NULL) {
```

Comment:
If someone could double check me on my output buffer length calculation here, that would be helpful.